app/policies/analytic_policy.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# User Policy
+class AnalyticPolicy < Struct.new(:user, :analytic)
```

Comment:
I'm a bit unclear about why the `Struct` is needed here.

Reply:
![screenshot from 2019-01-28 15-34-57](https://user-images.githubusercontent.com/13062617/51836773-6c9b2080-2312-11e9-9d4d-dd8bc3ded068.png)
I used because analytic does not have a model associated with it.

Reply:
@StlMaris123 This makes sense. Thanks for the explanation